src/auth-service/config/global/email-templates.js

```diff
@@ -78,8 +79,25 @@ const emailTemplates = {
                                             style="color: #667085; font-size: 14px; font-family: Inter; font-weight: 400; line-height: 20px; word-wrap: break-word;">This
                                             email was sent to</span>
                                         <span
-                                            style="color: #135DFF; font-size: 14px; font-family: Inter; font-weight: 400; line-height: 20px; word-wrap: break-word;">${email}</span>
+                                            style="color: #135DFF; font-size: 14px; font-family: Inter; font-weight: 400; line-height: 20px; word-wrap: break-word;">${email}<br/></span>
+                                            
+                                        ${this.EMAIL_UNSUBSCRIBE_CLAUSE({ optional })}
                                         <span
+                                            style="color: #667085; font-size: 14px; font-family: Inter; font-weight: 400; line-height: 20px; word-wrap: break-word;">
+                                            <br/>
+                                            ©2023 AirQo<br /><br />
+                                            Makerere University, Software Systems Centre, Block B, Level 3, College of
+                                            Computing and
+                                            Information Sciences, Plot 56 University Pool Road</span>
+                                    </td>
+                                </tr>
+                            </table>
+    `;
+  },
+
+  EMAIL_UNSUBSCRIBE_CLAUSE: function ({ optional = false }) {
```

Comment:
Thanks @BenjaminSsempala ,

Your current implementation of `EMAIL_UNSUBSCRIBE_CLAUSE` is almost perfect! Just a minor tweak could enhance its robustness. Consider adjusting the function signature to `EMAIL_UNSUBSCRIBE_CLAUSE: function ({ optional = false } = {}) { }`. This change ensures the function can handle cases where no arguments are passed, providing an extra layer of flexibility. Great work so far!